.circleci/config.yml

```diff
@@ -1,206 +1,133 @@
-            - 56:12:d8:11:e2:11:41:bc:f3:29:b7:ba:cc:d5:0c:5a
-      - run:
-          name: Clone Bichard7 Next
-          command: git clone --depth 1 git@github.com:ministryofjustice/bichard7-next.git ~/bichard7-next
+      - run: 
+          name: "Fetch images: <<parameters.IMAGES>>"
+          command: bash ./scripts/fetch-docker-images.sh <<parameters.IMAGES>>
 
-  run_postgres:
-    description: Run the PostgreSQL database
-    steps:
-      - run:
-          name: Spin up the PostgreSQL database
-          working_directory: ~/bichard7-next
-          command: make run-pg
 
-  run_mq:
-    description: Run MQ
+  build_message_forwarder:
     steps:
       - run:
-          name: Spin up MQ
-          working_directory: ~/bichard7-next
-          command: make run-activemq
+          name: Build message forwarder image
+          working_directory: ~/bichard7-next-core
+          command: docker-compose -f environment/docker-compose.yml build message-forwarder
 
-  run_audit_logging:
-    description: Run audit logging
+  build_audit_logging_images:
     steps:
       - run:
-          name: Run audit logging API
+          name: Build audit logging images
           working_directory: ~/bichard7-next-audit-logging
-          command: npm install && SKIP_PG=true SKIP_MQ=true npm run start-api
-          background: true
+          command: make build-api-server build-event-handler-server
 
-  use_nvm_version:
-    description: Install nvm and use configured node version
+  run_test_chunk:
+    parameters:
+      NEXTUI:
+        default: "false"
+        type: "string"
     steps:
       - run:
-          label: Use nvm for a specific node version...
+          name: Run test chunk
+          working_directory: ~/bichard7-next-tests
           command: |
-            set +e
-            export NVM_DIR="/opt/circleci/.nvm"
-            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-            nvm install
-            nvm use
+            CHUNK=$(circleci tests glob "features/**/*.feature" | circleci tests split --split-by=timings) \
+            NEXTUI=<<parameters.NEXTUI>> \
+            npm run test:chunk
 
-  checkout-tests:
+  ################### Services
+  start_bichard7_legacy:
     steps:
-      - run:
-          name: Git clone e2e tests
-          command: git clone --depth 1 https://github.com/ministryofjustice/bichard7-next-tests.git ~/bichard7-next-tests
-
-  checkout-audit-logging:
-    steps:
-      - run:
-          name: Git clone audit logging
-          command: git clone --depth 1 https://github.com/ministryofjustice/bichard7-next-audit-logging.git ~/bichard7-next-audit-logging
-
-  fetch_docker_image:
+      - fetch_images:
+          IMAGES: nodejs
+      - build_message_forwarder
+      - build_audit_logging_images
+      - run:
+          name: Start bichard7 legacy
+          working_directory: ~/bichard7-next-core
+          command: npm run all-legacy
+    
+  start_specific_core_services:
```

Comment:
I'll be pedantic here: Why core services? :D

Reply:
Fair point